AsyncDisplayKit/ASControlNode.h

```diff
@@ -119,7 +122,12 @@ typedef NS_OPTIONS(NSUInteger, ASControlState) {
   @param event The event which triggered these control actions. May be nil.
  */
 - (void)sendActionsForControlEvents:(ASControlNodeEvent)controlEvents withEvent:(nullable UIEvent *)event;
-
+#if TARGET_OS_TV
```

Comment:
@appleguy Since tvOS handles all it's display states internally, there isn't a similar method. The closest to this would be `setNeedsFocusUpdate` but this implies a different action. Maybe we can make it more descriptive, such as `setDefaultFocusAppearance`